docs/features/dynamic-analysis.work.md

```diff
@@ -0,0 +1,14 @@
+Dynamic Analysis
+===============
+
+This feature enables instrumentation of binaries to collect information for dynamic analysis.
+
+--------------------
+
+TODO:
+
+- Generate code to compute MVIDs less often than once per invoked method.
```

Comment:
Oh. I thought it involved writing some actual markup. Even I can type '[' and ']' characters...